chaos-monkey-spring-boot/src/test/java/de/codecentric/spring/boot/chaos/monkey/configuration/ChaosMonkeyRequestScopeSettingsTest.java

```diff
@@ -16,23 +16,23 @@
 
 package de.codecentric.spring.boot.chaos.monkey.configuration;
 
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
 
-import static junit.framework.TestCase.assertTrue;
```

Comment:
Replacing this import with `org.junit.Assert.assertTrue` is a suggestion by IntelliJ.